### PR TITLE
update dead links

### PR DESCRIPTION
**Pull request by carlad**
**Proposed changes**:
These are minor changes to documentation that fix a few dead links following the release of Rasa Enterprise 1.0

### CODE REVIEW

File: docs/docs/tuning-your-model.mdx [hunk 1]
```diff
@@ -385,8 +385,8 @@ pipeline:
 ### Accessing Diagnostic Data
 
 To gain a better understanding of what your models do, you can access intermediate results of the prediction process.
-To do this, you need to access the `diagnostic_data` field of the [Message](./reference/rasa/shared/nlu/training_data/message.md#message-objects)
-and [Prediction](./reference/rasa/core/policies/policy.md#policyprediction-objects) objects, which contain
+To do this, you need to access the `diagnostic_data` field of the [Message](https://rasa.com/docs/rasa/reference/rasa/shared/nlu/training_data/message)
+and [Prediction](https://rasa.com/docs/rasa/reference/rasa/core/policies/policy#policyprediction-objects) objects, which contain
```
Comment (m-vdb) [inline docs/docs/tuning-your-model.mdx:389]: same as above, what's wrong?

File: docs/docs/training-data-importers.mdx [hunk 2]
```diff
@@ -145,7 +145,7 @@ their training data will be combined.
 ## Writing a Custom Importer
 
 If you are writing a custom importer, this importer has to implement the interface of
-[`TrainingDataImporter`](reference/rasa/shared/importers/importer.md#trainingdataimporter-objects):
+[`TrainingDataImporter`](https://rasa.com/docs/rasa/reference/rasa/shared/importers/importer):
```
Comment (m-vdb) [inline docs/docs/training-data-importers.mdx:148]: what's wrong with this link? the local URI should work 🤔

**Review by m-vdb — CHANGES_REQUESTED** [review]

**Comment by m-vdb** [comment]
@carlad I don't think that this PR fixes the error you had [here](https://github.com/RasaHQ/rasa/runs/4583863881?check_suite_focus=true#step:11:166) 🤔

**Comment by carlad** [comment]
> @carlad I don't think that this PR fixes the error you had [here](https://github.com/RasaHQ/rasa/runs/4583863881?check_suite_focus=true#step:11:166) 🤔

That error doesn't appear on `main` as the `docs/how-to-deploy.mdx` page no longer exists.

**Pull request closed without merging**